components/navigation/NavBar.js

```diff
@@ -53,6 +52,8 @@ export default function NavBar ({
             <svg className="h-6 w-6" stroke="currentColor" fill="none" viewBox="0 0 24 24">
               <title>Menu</title>
               <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M4 6h16M4 12h16M4 18h16" />
+              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M4 6h16M4 12h16M4 18h16"/>
```

Comment:
The 2 lines added here are not needed as they overwrite the existence of 3 parallel lines of Menu. You can remove this.

Reply:
I've removed them.